artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/FileLockNodeManager.java

```diff
@@ -108,7 +108,16 @@ public synchronized void start() throws Exception {
    protected synchronized void setUpServerLockFile() throws IOException {
       super.setUpServerLockFile();
 
-      for (int i = 0; i < 3; i++) {
+      if (lockChannels[0] == null || !lockChannels[0].isOpen()) {
+         File fileLock = newFile(NodeManager.SERVER_LOCK_NAME);
+         if (!fileLock.exists()) {
+            fileLock.createNewFile();
```

Comment:
what if return false, because already exists?

Reply:
If the `if` returns false then that means `lockChannels[0]` is not equal to `null` and it's open which means we don't need to create the file or get the channel. It's the same basic logic that was already in place. I didn't change the semantics on that part.

Reply:
I mean createNewFile: it return a boolean

Reply:
if file already existed, we just use the previous.. that's the semantic in place

Reply:
:+1: